crud-frontend/src/components/login/login.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { createClient } from "@supabase/supabase-js";
+
+const supabase = createClient(process.env.REACT_APP_SUPABASE_URL, process.env.REACT_APP_SUPABASE_PUBLIC_ANON_KEY, {
+	auth: {
```

Comment:
Please add instructions to the README about what env vars are necessary, where to find them, and where to set them like an `.env` file.